pkg/http/header/header.go

```diff
@@ -58,6 +58,8 @@ const (
 	// KubeletProbeKey is the name of the header supplied by kubelet
 	// probes.  Istio with mTLS rewrites probes, but their probes pass a
 	// different user-agent.  So we augment the probes with this header.
+	//
+	// Deprecated: use knative.dev/networking/pkg/http/header.UserAgentKey
```

Comment:
I'd be more specific in this deprecation notice and say we want to detect probes with `header.UserAgentKey` and `KubeProbeUAPrefix`

Reply:
Made changes. Let me know if the text makes sense